pkg/clustermesh/common/clustermesh.go

```diff
@@ -178,8 +178,10 @@ func (cm *clusterMesh) add(name, path string) {
 	}
 
 	if err := types.ValidateClusterName(name); err != nil {
-		log.WithField(fieldClusterName, name).WithError(err).
-			Error("Remote cluster name is invalid. The connection will be forbidden starting from Cilium v1.17")
+		log.WithField(fieldClusterName, name).
+			WithError(fmt.Errorf("invalid cluster name: %w", err)).
```

Comment:
Should the formatting rule be repeated here?

Reply:
I personally think it is useful to have the formatting rule explicitly repeated here, so that users can see why the connection got rejected. With the current implementation, it is included as part of the error returned by `ValidateClusterName` and printed here.